source/optional_features/block_prepaid_cards/index.html.md.erb

```diff
@@ -0,0 +1,17 @@
+---
+title: Block prepaid cards
+weight: 140
+---
+
+# Block prepaid cards
+
+You can block users from paying with prepaid cards. You should only do this if users are making fraudulent payments with prepaid cards.
+
+You cannot block most Mastercard prepaid cards, because we usually cannot tell if Mastercard cards are prepaid or not.
+
+If you want to block Visa prepaid cards, you must register with Visa to get permission first. To register, either:
+
+- [contact us](/support_contact_and_more_information/#contact-us) if you use Stripe
+- contact your Payment Service Provider
+
+[Contact us](/support_contact_and_more_information/#contact-us) to block prepaid cards on your account.
```

Comment:
Being right at the end, I worry this step could get lost and a service might think they only have to contact their PSP (or us in the case of Stripe).

Reply:
I can't see another way of structuring this unfortunately - but as above I'll see what our 2ier thinks.